packages/nimble-components/src/rich-text-editor/template.ts

```diff
@@ -1,6 +1,71 @@
-import { html } from '@microsoft/fast-element';
+import { html, ref } from '@microsoft/fast-element';
 import type { RichTextEditor } from '.';
+import { toolbarTag } from '../toolbar';
+import { toggleButtonTag } from '../toggle-button';
+import { iconBoldBTag } from '../icons/bold-b';
+import { iconItalicITag } from '../icons/italic-i';
+import { iconNumberListTag } from '../icons/number-list';
+import { iconListTag } from '../icons/list';
 
 export const template = html<RichTextEditor>`
-    <template> Rich Text Editor here. </template>
+    <template>
+        <div class="container">
+            <section ${ref('editor')} class="editor"></section>
+            <section class="footer-section">
+                <${toolbarTag}>
+                    <${toggleButtonTag} ${ref('bold')}
+                        content-hidden
+                        appearance="ghost"
+                        slot="start"
+                        class="bold"
+                        @click=${x => x.boldButtonClickHandler()}
+                        @keydown=${(x, c) => x.boldButtonKeyDownHandler(
+        c.event as KeyboardEvent
+    )}
+                        >
+                        <${iconBoldBTag} slot="start"></${iconBoldBTag}>
+                    </${toggleButtonTag}>
+                    <${toggleButtonTag} ${ref('italics')}
+                        content-hidden
+                        appearance="ghost"
+                        slot="start"
+                        class="italics"
```

Comment:
await: Is there a possibility to use `repeat` in the template and keep these buttons in an array view model and have the properties like `ref`, `class`, etc. declared in `EditorButton` and defined in corresponding button child classes? 

By this way you don't have to create `boldButtonClickHandler`, `italicsButtonClickHandler`, etc. for each button in `index.ts`. Instead you can just iterate with `repeat` and call just `clickHandler` from the button classes

Reply:
I have updated the code as suggested.